pkg/config/auth.go

```diff
@@ -192,6 +194,8 @@ func (a *auth) ToUpdateAuthConfigBody() v1API.UpdateAuthConfigBody {
 		SecurityManualLinkingEnabled:      &a.EnableManualLinking,
 		DisableSignup:                     cast.Ptr(!a.EnableSignup),
 		ExternalAnonymousUsersEnabled:     &a.EnableAnonymousSignIns,
+		PasswordMinLength:                 cast.UintToIntPtr(&a.MinimumPasswordLength),
+		PasswordRequiredCharacters:        (*v1API.UpdateAuthConfigBodyPasswordRequiredCharacters)(&a.PasswordRequirements),
```

Comment:
Also need to update `fromRemoteAuthConfig` for the reverse mapping.

Reply:
@sweatybridge how can I find out what value `fromRemoteAuthConfig` will return? because I then need to convert these from `regex` or a `string` to match the enum map I'm using in the CLI.

Reply:
It's ok to cast the response field to enum.

Reply:
@sweatybridge How can I test the fromRemoteAuthConfig functionality?

Reply:
I've added the tests as an example.